### ISSUE TITLE
Icons are rendered small when using powerlevel9k

### ISSUE THREAD

**Issue by nivpgir** [kovidgoyal/kitty#220]
I'm using powerlevel9k with vanilla zsh on my setup, and as the title describes
the icons in prompt are being drawn smaller than expected, see image for example:

![termite-vs-kitty-icons](https://user-images.githubusercontent.com/10796080/33774851-434ad71c-dc45-11e7-998b-2c876d263cf8.png)

running arch linux 4.14.3-1-ARCH.

using the default kitty.conf with the exception of my fonts being:
font_family      Source Code Pro # monospace
italic_font      Source Code Pro Italic # auto
bold_font        Source Code Pro Bold # auto
bold_italic_font Source Code Pro Bold Italic # auto

and some color changes

**Comment by egmontkob** [comment]
Based on the positioning of the house (presumably U+1F3E0) / folder (U+1F4C1 or U+1F4C2) icons and the subsequent tilda character, I assume you're still using Unicode 8.0 or older. Unicode 9.0 changed their width from narrow to wide (double cell) (see "N" in [8.0](http://ftp.unicode.org/Public/8.0.0/ucd/EastAsianWidth.txt) vs. "W" in [9.0](http://ftp.unicode.org/Public/9.0.0/ucd/EastAsianWidth.txt)).

For Termite (VTE) the Unicode version upgrade is introduced by glib-2.50.1. For Kitty I _guess_ that it's glibc that matters which upgraded in version 2.26.

See https://bugzilla.gnome.org/show_bug.cgi?id=772812 and follow the links from there to understand the mess this change created.

When a glyph is wider than its designated cell, VTE overflows it to the right. Apparently Kitty rescales the glyph to fit in its cell; Kitty's author will confirm whether I'm right here. This is not a bug in Kitty nor in VTE, just a different approach.

Upgrading to Unicode 9.0 will mitigate this problem as there'll be more room for these glyphs and hence I'm quite certain Kitty won't scale it down.

**Comment by nivpgir** [comment]
@egmontkob 
I see... sorry for alarming...

if I may ask, how can I upgrade to unicode 9.0? 
after I read your thread I thought upgrading to glibc 2.26 would be enough, but the problem persists... 

I would appreciate an answer very much,
and in any case, thank you.

**Comment by egmontkob** [comment]
I'm a VTE guy; let's wait for Kitty's author to explain what is exactly going on here with Kitty.

**Comment by B4rtware** [comment]
I have the same problem.
![image](https://user-images.githubusercontent.com/34386047/33790485-d7705d28-dc80-11e7-898a-837236c287b7.png)
But they aren't even correctly aligned in the centre. I have tried to use a Mono suffixed font but this 
changed nothing. Even the wcw setting which the developer mentioned in another closed issue regarding powerline did not work.

**Comment by kovidgoyal** [comment]
https://github.com/kovidgoyal/kitty#some-special-symbols-are-rendered-smalltruncated-in-kitty

**Comment by nivpgir** [comment]
ok, so the state for me is such:

I've updated to glibc 2.26 which from the thread linked by egmontkob is supposed to fix this issue, but icons are still small.

I tried using use the fix with 'use_system_wc_width no' in kitty.conf, while aware of the possible issues, just to see if it fixes the problem, but icons remain small.

tried reinstalling kitty from aur, in the hopes that it will recompile with gcc 2.26, but still no joy.

I just started using kitty, so i'm no expert but maybe this means that wcwidth is not the problem, and there is something else wrong with the double celled icons?
